src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +104,13 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate today = LocalDate.now();
+        if (someDate.isAfter(today)) {
+            return String.format("%s is after %s", someDate.toString(), today.toString());
+        } else if (someDate.isBefore(today)) {
+            return String.format("%s is before %s", someDate.toString(), today.toString());
+        }
+        return String.format("%s is today", someDate.toString());
```

Comment:
if you wish you can make this more compact using `String.format()` and the ternary operator

Reply:
done